change/@microsoft-teams-js-891e4235-67b9-4b5a-b0f7-d503a900afca.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add isSupported to stageView",
```

Comment:
Please update the comment to the following: "Added `isSupported` to `stageView`" -- please enclose "isSupported" and "stageView" in backticks and use past tense. (These guidelines are also documented in the CONTRIBUTING.md file in the monorepo root.) Thanks!